vizro-core/docs/pages/user-guides/assets.md

```diff
@@ -147,42 +155,47 @@ Suppose you want to hide the page title on one page only. Here's how you can ach
 
 
 ## Overwrite CSS properties in selective components
-To overwrite CSS properties of selective components, pass an ID to the relevant component and target the right CSS property.
+To adjust CSS properties for specific components, like altering the appearance of a selected [Card][vizro.models.Card] rather than all Cards,
+you'll need to target the correct CSS selector based on the component hierarchy.
```

Comment:
```suggestion
To adjust CSS properties for specific components, like altering the appearance of a selected [`Card`][vizro.models.Card] rather than all `Card`s,
you need to use a CSS selector that targets the right component.
```